docs/source/extensions/plugins.rst

```diff
@@ -18,6 +18,17 @@ There are three types of Plugins in Lightning with different responsibilities:
 - CheckpointIO Plugins
 - Cluster Environments
 
+You can make the Trainer use one or multiple plugins by adding it to the ``plugins`` argument like so:
```

Comment:
In this plugin session, should we mention that if no plugin is passed in, we will choose for user?